src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncSpillTests.cs

```diff
@@ -3290,5 +3290,55 @@ public static void Main()
                 Diagnostic(ErrorCode.ERR_RefReturningCallAndAwait, "P").WithArguments("C.P.get").WithLocation(18, 28)
                 );
         }
+
+        [Fact]
+        [WorkItem(27831, "https://github.com/dotnet/roslyn/issues/27831")]
```

Comment:
Is this test related to the bug or the code change? If not, consider removing `[WorkItem]`.